src/components/CardMovie.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-container>
+    <div v-for="movie in movies" :key="movie.id" class="card-movie">
```

Comment:
1. Rename the current component from "CardComponent" to "MoviesList" (iterator should be here)
2. Extract new "CardMovie" component for *single* movie card (just card for one movie should be here)